lte/gateway/python/magma/pipelined/app/enforcement.py

```diff
@@ -151,7 +151,9 @@ def _get_default_flow_msgs(self, datapath) -> DefaultMsgsMap:
 
         return {self.tbl_num: [msg]}
 
-    def _get_rule_match_flow_msgs(self, imsi, msisdn: bytes, uplink_tunnel: int, ip_addr, apn_ambr, rule, version):
+    def _get_rule_match_flow_msgs(self, imsi, msisdn: bytes, uplink_tunnel: int,
+                                  ip_addr, apn_ambr, rule, version,
+                                  local_f_teid_ng=0):
```

Comment:
It looks to me like we're always passing this around, lets just make this a required arg (remove =0)

Reply:
done.